deploy/services-demo/conf/brig.demo-docker.yaml

```diff
@@ -88,7 +99,7 @@ optSettings:
   setActivationTimeout: 1209600     # 1 day
   setTeamInvitationTimeout: 1814400 # 21 days
   setUserMaxConnections: 1000
-  setCookieDomain: brig
```

Comment:
I am curious about this one though: if we also run api-simulations as a docker image, this won't work, right?

Reply:
I'm not sure; I can revert it anyways since the more useful one is the local demo anyways.